docs/jwst/jump/arguments.rst

```diff
@@ -38,4 +38,14 @@ The ``jump`` step has five optional arguments that can be set by the user:
   not have its neighbors flagged. The goal is to prevent flagging jumps that would be too
   small to significantly affect the slope determination.  The default value is 10.
 
+*  ``--after_jump_flag_dn1``: A floating point value in units of DN that gives the
```

Comment:
update this to be in sync with what is in stcal now, the same docstrings there and e instead of dn

Reply:
User gives these values in units of DN. They get converted internally to electrons for use in the stcal functions.